python/cuda_cccl/cuda/cccl/parallel/experimental/iterators/_iterators.py

```diff
@@ -99,25 +98,21 @@ def __init__(
         ----------
         cvalue
           A ctypes type representing the object pointed to by the iterator.
-        numba_type
-          A numba type representing the type of the input to the advance
-          and dereference functions.
         state_type
-          A numba type of the iterator state.
```

Comment:
The `numba_type` is _always_ a pointer to `state_type`, so one of these had to go.

Reply:
We should probably just rename `numba_type` to `state_ptr_type`.